plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHABase.java

```diff
@@ -57,6 +58,28 @@ public NfsStoragePool(String poolUUID, String poolIp, String poolSourcePath, Str
         }
     }
 
+    public static class RbdStoragePool {
+        String _poolUUID;
```

Comment:
IMHO, using strict camel case (non consecutive capitals/treat acronyms as normal words) would improve readability; with this approach, the variable name would be `poolUuid`. Also, this approach is already being applied in the class name: it is `RbdStoragePool` instead of  `RBDStoragePool`.